src/transaction/log_page_buffer.c

```diff
@@ -7049,7 +7049,8 @@ logpb_checkpoint (THREAD_ENTRY * thread_p)
 
   LOG_CS_ENTER (thread_p);
 
-  assert (LSA_LE (&oldest_unflushed_lsa, &new_chkpt_lsa));
+  // assert is in line with what happens in function pgbuf_flush_seq_list where value is collected
+  assert (oldest_unflushed_lsa.is_null ());
```

Comment:
@wrlawodms 
The reasoning for the assert change is explained in http://jira.cubrid.org/browse/CBRD-24057 .
Once that issue is accepted as a solution or rejected, the assert here can either be, respectively, eiter removed altogether or reverted to the previous form.

Reply:
I have left my comment on it in the CBRD issue. Thank you.